frontend/src/Plugin.js

```diff
@@ -143,6 +144,8 @@ class ShopgateAnalytics extends SgTrackingPlugin {
         sdkData.meta = rawData.meta;
       }
 
+      const checkoutCompletedEvent = new HttpRequest();
+      checkoutCompletedEvent.setMethod('POST').setPayload(sdkData).dispatch();
```

Comment:
isn't the name of the event now missing here?
+ it should be a different name

Reply:
Updated name of the event. Must've got the name tongue twisted.

Reply:
Originally misunderstood. Fixed object structure and added content type function for HttpRequest command